src/components/map/layers/new/NewRoutePathLayer.tsx

```diff
@@ -14,37 +14,58 @@ interface IRoutePathLayerProps {
 @observer
 export default class RoutePathLayer extends Component<IRoutePathLayerProps> {
 
-    private getMarkerHtml = () => {
-        return `<div class="${classnames(s.nodeBase, s.newRoutePathMarker)}" />`;
+    private getMarkerHtml = (isNeighbor: boolean) => {
+        return `<div class="${classnames(
+            s.nodeBase,
+            isNeighbor ? s.newRoutePathMarkerNeighbor : s.newRoutePathMarker,
+        )}" />`;
     }
 
-    private getIcon() {
+    private getIcon(isNeighbor: boolean) {
         const divIconOptions : L.DivIconOptions = {
-            html: this.getMarkerHtml(),
+            html: this.getMarkerHtml(isNeighbor),
             className: s.node,
         };
 
         return new L.DivIcon(divIconOptions);
-
     }
 
     private renderNodes() {
         return this.props.newRoutePathStore!.nodes.map((node, index) => {
             const latLng = L.latLng(node.coordinates.lat, node.coordinates.lon);
             return (
                 <Marker
-                    icon={this.getIcon()}
+                    icon={this.getIcon(false)}
```

Comment:
Could try to pass object instead.

this.getIcon({isNeightbor: false})
public getIcon({isNeighbor: boolean})
public getIcon(options)
const { isNeighbor } = options